addons/sale/views/sale_portal_templates.xml

```diff
@@ -429,6 +429,10 @@
                             t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'/>
                     </div>
                 </div>
+                <div t-if="sale_order.client_order_ref">
+                    <strong>Your Reference:</strong>
+                    <t t-out="sale_order.client_order_ref"/>
```

Comment:
Hello :wave: For security reason, I would use `t-esc` instead of `t-out`. if I'm not mistaken: `t-esc` will escape all HTML that could be stored in the field while `t-out` won't. If the field `client_order_ref` does not contain HTML coming from a trusted source, we should rather use `t-esc` :smile:

Reply:
I'm sorry. I misunderstood something about `t-esc` and `t-out`. `t-esc` is actually deprecated https://www.odoo.com/documentation/master/developer/reference/frontend/qweb.html#deprecated-output-directives

> esc
An alias for out, would originally HTML-escape its input. Not yet formally deprecated as the only difference between out and esc is that the latter is a bit unclear / incorrect.

May I ask you to put `t-out` back ? You can green the task once it's done :smiley:

Reply:
Okay :)